tests/test_level3/test_not.py

```diff
@@ -47,3 +48,10 @@ def test_not_case(self):
             ["0", "2", "3", "4", "5", "6", "pre"],
             flags=util.HTML
         )
+
+    def test_none_inputs(self):
+        """Test weird inputs."""
+
+        soup = BS('<span foo-"something">text</span>', 'html.parser')
```

Comment:
Is the minus sign intended? or a typo？

Reply:
Yeah, a typo. I had tested the behavior locally, so I'm sure the case works, but this test was a typo. It is now fixed on the main branch and the test works (just for the correct reasons now 🙂).